### DISCUSSION TITLE
Windows installation issues

### DISCUSSION THREAD

**Discussion by kalimuthu123** [apache/superset#14273] [General]
Traceback (most recent call last):
  File "D:\bitool\venv\lib\site-packages\superset\views\core.py", line 2603, in _sql_json_sync
    with utils.timeout(seconds=timeout, error_message=timeout_msg):
  File "D:\bitool\venv\lib\site-packages\superset\utils\core.py", line 593, in __enter__
    signal.signal(signal.SIGALRM, self.handle_timeout)
AttributeError: module 'signal' has no attribute 'SIGALRM'

the issue came around the tool after windows installation

**Comment by kalimuthu123** [comment]
https://github.com/apache/incubator-superset/issues/9992

**Comment by dpgaspar** [comment]
@kalimuthu123,

This is probably related to https://stackoverflow.com/questions/52779920/why-is-signal-sigalrm-not-working-in-python-on-windows. Note that windows is not a supported platform for superset

**Comment by villebro** [comment]
In theory it would be easy to bypass this by adding a `hasattr(signal, "SIGALRM")` to the if statement, but I'm not sure running Superset without timeouts is a good idea, either. I'd be curious to hear what others have done to get Superset running on Windows.

**Comment by Xiaojun-Pan** [comment]
modify  core.py  code

def __exit__(  # pylint: disable=redefined-outer-name,unused-variable,redefined-builtin
        self, type: Any, value: Any, traceback: TracebackType
    ) -> None:
        try:
            pass
        except ValueError as ex:
            logger.warning("timeout can't be used in the current context")
            logger.exception(ex)

**Comment by junlincc** [comment]
centralizing Windows related questions here. 

https://github.com/apache/superset/issues/13532
While using superset for creating dashboards and some useful charts we are getting the following error after 1 minute.

We’re having trouble loading this visualization. Queries are set to timeout after 60 seconds.
This may be triggered by:
Issue 1000 - The dataset is too large to query.
Issue 1001 - The database is under an unusual load.

Any alternative solution or any option to increase this time out?

Note: Currently, we are using Ubuntu tool available on Microsoft store to deploy and run Superset server. So not able to directly access and update config.py file.
